react/features/screen-share/middleware.js

```diff
@@ -26,6 +31,19 @@ MiddlewareRegistry.register(store => next => action => {
         _setScreenshareCaptureFps(store, captureFrameRate);
         break;
     }
+
+    case SET_SCREEN_AUDIO_SHARE_STATE: {
+        const { isSharingAudio } = action;
+        const { participantId } = state['features/large-video'];
```

Comment:
Why do we take the `participantId` from `large-video`? How is audio sharing related to large video?